source/_components/media_player.nad7050.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "NAD D 7050"
+description: "Instructions how to integrate the NAD D 7050 digital amplifier into Home Assistant."
+date: 2017-04-01 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nad.png
+ha_category: Media Player
+ha_release: 0.4x
+ha_iot_class: "Local Polling"
+---
+
+
+The `nad7050` platform allows you to control a [NAD D 7050](https://nadelectronics.com/product/d-7050-direct-digital-network-amplifier/) digital amplifier from Home Assistant.
+
+To add a NAD receiver to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: nad7050
+    name: NAD D 7050
+    host: 192.168.0.112
+    min_volume: -60
+    max_volume: -20
+    volume_step: 5
+```
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.